addons/map_gestures/functions/fnc_receiverInit.sqf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: Dslyecxi, MikeMatrix
+ * Initializes the receiver and hooks it to the Draw event of the map.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_map_gestures_fnc_receiverInit
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+{
+    if (isPlayer _x) then {
+        _nameSane = [name _x] call FUNC(sanitizeName);
+        missionNamespace setVariable [format [QGVAR(%1_DrawPos), _nameSane], [1, 1, 1]];
+    };
+    nil
+} count allUnits;
```

Comment:
1. Upon receiving this default value is also given. Is there a specific reason to loop through all units and assign default values upon init as well?
2. This will not work for any JIP players.